040-Data-operations/160-image-transformations.mdx

```diff
@@ -7,13 +7,19 @@ slug: sdk/image-transformations
 published: true
 ---
 
-For the very common case of storing images, Xata offers a set of transformations that can be applied in real-time while retrieving images through [access URLs](/docs/sdk/file-attachments#file-access-urls). Transformations are specified in the URL and their result is cached by the CDN. Both public and signed URLs support transformations. In the URL the transformation specification is a comma separated list of individual transformations following the `/transform/` segment.
+Xata supports transformations for images stored in Xata's `File` and `File[]` [column types](/docs/sdk/file-attachments).
 
-Original image URL:
+Transformations are specified by modifying the `File`'s URL by adding `/transform/height=100,width=200/{fileId}`
+
+## Example
+
+(Vanity idea: gif which animates pulling the url apart and inserting the transform. And also animates the actual image :fire:. Or it could be react code not a gif. Might be too distracting)
+
```

Comment:
Vanity idea: gif which animates pulling the url apart and inserting the transform. And also animates the actual image :fire:. Or it could be react code not a gif. Might be too distracting